runscanner/src/main/java/uk/ac/bbsrc/tgac/miso/runscanner/processors/StandardIllumina.java

```diff
@@ -43,10 +45,15 @@ public StandardIllumina(Builder builder) {
   }
 
   @Override
+  public Stream<File> getRunsFromRoot(File root) {
+    return Arrays.stream(root.listFiles(File::isDirectory));
+  }
+
+  @Override
   public NotificationDto process(File runDirectory, TimeZone tz) throws IOException {
     // Call the C++ program to do the real work and write a notification DTO to standard output. The C++ object has no direct binding to the
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)